apps/cookbook/src/app/examples/chart-example-stock/chart-example-stock.component.html

```diff
@@ -1,10 +1,10 @@
 <kirby-card>
-  <kirby-card-header [title]="'Highstock Chart'"></kirby-card-header>
-  <kirby-highstock-chart
+  <kirby-card-header [title]="'Stock Chart'"></kirby-card-header>
```

Comment:
Consider replacing with [One-time string initialization](https://angular.io/guide/template-syntax#one-time-string-initialization)
```suggestion
  <kirby-card-header title="Stock Chart"></kirby-card-header>
```

Reply:
Done